### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by LeahWinters**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Migration 
- [X] Styling
- [ ] Testing

### Commit Emojis:
:art: CSS modifications/updates.
:crystal_ball: JS modifications/updates.

### Detailed Description:

- Completed by whole FE team: Liliana, John and Léah. Found in the first commit that they were both co-authors.
- Created the home page that contains a header and a spot to view all of the destinations to take a voyage to.
- Added styling with animation to the whole page

### Why is this change required? What problem does it solve?

So that when the user first loads the page, they see an awesome looking application

### Were there any challenges that arose while implementing this feature? If so, how were the addressed?

Right now the destination cards are hard coded in. This will be fixed once we have data to work with from the backend team.

### Files modified:
All files were created and worked on.

### CODE REVIEW

File: src/Components/Destinations/Destinations.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  return (
+    <div className='destinations'>
+      <Slider  {...settings}>
+        <DestinationCard />
+        <DestinationCard />
+        <DestinationCard />
+        <DestinationCard />
+      </Slider>
+    </div>
```
Comment (lilianaweimer) [inline src/Components/Destinations/Destinations.js:28]: Great carousel! Can't wait for this to be dynamic.

**Review by lilianaweimer — APPROVED** [review]
This looks great! 🥳

**Comment by infamouskeyduster** [comment]
All commits co-authored by @LeahWinters @lilianaweimer @infamouskeyduster BUT we could not figure out how to interactive rebase and add co-authors to our commits ☹️ ; When paired programming in the future, we will make sure to add co-authors at the time of the original commit!

**Review by infamouskeyduster — APPROVED** [review]
All team members present for coding, no further review needed at this time!

**Pull request merged**